identity-apps-core/apps/x509-certificate-authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -16,6 +16,19 @@
 # under the License.
 #
 
+# <!-- Start of Common Translations -->
+# EDITABLE=true,SCREEN="common",MULTI_LINE=true
+copyright=© {{currentYear}} WSO2 LLC.
+# EDITABLE=true,SCREEN="common",MULTI_LINE=false
+site.title=WSO2 Identity Server
+# EDITABLE=false,SCREEN="common",MULTI_LINE=false
+powered.by=Powered by
+# EDITABLE=true,SCREEN="common",MULTI_LINE=false
+privacy.policy=Privacy Policy
+# EDITABLE=true,SCREEN="common",MULTI_LINE=false
```

Comment:
Do we need to keep these comments anymore?

Reply:
The branding text customization feature relies on these comments.